wordpress-mobile/WordPress-Android
# Unified comments details feature flag and basic activity

This PR adds a base for Unified Comments Details - feature flag and basic activity with viewpager and couple of empty fragments.

To test:

- Install vanilla build.
- Navigate to the site comments list.
- Tap on any comment to open comment details.
- Confirm that the comment details are opened.

----
- Install wasabi build.
- Navigate to the site comments list.
- Tap on any comment to open comment details.
- Confirm that the new comment details scaffolding are opened.

## Regression Notes
1. Potential unintended areas of impact
- The feature might become available in vanila build.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
- Manual testing.

3. What automated tests I added (or what prevented me from doing so)
- We need better integration with the comment list VM, which will come later.

PR submission checklist:
